polling_stations/apps/data_collection/management/commands/import_warwick.py

```diff
@@ -3,7 +3,115 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E07000222"
-    addresses_name = "parl.2017-06-08/Version 1/Warwick Democracy_Club__08June2017.tsv"
-    stations_name = "parl.2017-06-08/Version 1/Warwick Democracy_Club__08June2017.tsv"
-    elections = ["parl.2017-06-08"]
+    addresses_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Warwick.tsv"
+    stations_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Warwick.tsv"
+    elections = ["local.2019-05-02"]
     csv_delimiter = "\t"
+
+    def station_record_to_dict(self, record):
+
+        if record.polling_place_id == "7562":
+            record = record._replace(polling_place_postcode="CV35 8JE")
+
+        # Wasperton Village Hall, Wasperton
+        if record.polling_place_id == "7232":
+            record = record._replace(polling_place_postcode="CV35 8EB")
+        # St. Paul's Church Hall, Leicester Street, Royal Leamington Spa
+        if record.polling_place_id in ["7375", "7392"]:
+            record = record._replace(polling_place_postcode="CV32 4TE")
+        # Sherbourne Village Hall, Sherbourne
+        if record.polling_place_id == "7225":
+            record = record._replace(polling_place_postcode="CV35 8AN")
+        # Lillington Nursery, Lillington Nursery & Primary School, Grange Road Entrance, Off Pound Lane, Lillington, Royal Leamington Spa
+        if record.polling_place_id == "7395":
+            record = record._replace(polling_place_postcode="CV32 7AG")
+        # Lowsonford Village Hall, Lowsonford
+        if record.polling_place_id == "7280":
+            record = record._replace(polling_place_postcode="CV35 8JE")
+        # Church of Jesus Christ Of Latter Day Saints, Saltisford/Birmingham Road/A425, Warwick
+        if record.polling_place_id in ["7526", "7476", "7479"]:
+            record = record._replace(polling_place_postcode="CV34 4TT")
```

Comment:
If these are corrections, can we confirm them with the council first (stick them in a trello card for Peter to follow up). Usually our suggested corrections are right, but occasionally they'll come back and request we use a slightly different postcode to the one we suggest.

Reply:
That's now here: https://trello.com/c/EKhqevnh